common/app/controllers/EmailSignupController.scala

```diff
@@ -71,6 +76,14 @@ class EmailFormService(wsClient: WSClient) extends LazyLogging with RemoteAddres
         .addHttpHeaders(headers: _*)
         .post(consentMailerPayload)
     }
+  }
+
+  private def serviceUrl(form: EmailForm): String = {
+    if (NewslettersRemoveConfirmationStep.isSwitchedOn && !form.emailConfirmation) {
+      return s"${Configuration.id.apiRoot}/consent-signup"
```

Comment:
please can we avoid using return here as per my comment in the original PR 
an if/else is more suitable I believe